### PR TITLE
Backport ob zh docs

### PR DESCRIPTION
**Pull request by leonardBang**

### CODE REVIEW

**Comment by rookiegao** [comment]
https://github.com/ververica/flink-cdc-connectors/blob/release-2.3/docs/content/connectors/oceanbase-cdc.md
The number of lines in the corresponding document does not match

**Comment by leonardBang** [comment]
> https://github.com/ververica/flink-cdc-connectors/blob/release-2.3/docs/content/connectors/oceanbase-cdc.md The number of lines in the corresponding document does not match

Hi, @rookiegao number of lines? I did not catch your meaning

**Comment by rookiegao** [comment]
> > https://github.com/ververica/flink-cdc-connectors/blob/release-2.3/docs/content/connectors/oceanbase-cdc.md The number of lines in the corresponding document does not match
> 
> Hi, @rookiegao number of lines? I did not catch your meaning

image


image

File: docs/content/connectors/oceanbase-cdc(ZH).md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+## 下载 SQL 客户端 JAR 包
+
+点击 [flink-sql-connector-oceanbase-cdc-2.3.0.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-oceanbase-cdc/2.3.0/flink-sql-connector-oceanbase-cdc-2.3.0.jar) 下载 JAR 包至 `<FLINK_HOME>/lib/`.
+
+> **说明：**
+>
+> 下载链接仅适用于稳定发行版本。
+
+`flink-sql-connector-oceanbase-cdc-XXX-SNAPSHOT` 快照版本与开发分支的版本对应。要使用快照版本，您必须自行下载并编译源代码。推荐使用稳定发行版本，例如 `flink-sql-connector-oceanbase-cdc-2.3.0.jar`。您可以在 Maven 中央仓库中找到使用稳定发行版本。
```
Comment (rookiegao) [inline docs/content/connectors/oceanbase-cdc(ZH).md:26]: Users need to download the source code and compile the corresponding jar. Users should use the released version, such as [flink-sql-connector-oceanbase-cdc-2.3.0.jar](https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-oceanbase-cdc), the released version will be available in the Maven central warehouse.

File: docs/content/connectors/oceanbase-cdc(ZH).md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+      env.execute("Print OceanBase Snapshot + Change Events");
+   }
+}
+```
+
+## 数据类型映射
+
+当启动模式不是 `INITIAL` 时，连接器无法获得一个列的精度和比例。为兼容不同的启动模式，连接器不会将一个不同精度的 OceanBase 类型映射到不同的FLink 类型。例如，`BOOLEAN`、`TINYINT(1)` 或 `BIT(1)` 均可以转换成 `BOOLEAN`。在 OceanBase 数据库中，`BOOLEAN` 等同于 `TINYINT(1)`，所以 `BOOLEAN` 和 `TINYINT` 类型的列在 Flink 中会被映射为 `TINYINT`，而 `BIT(1)` 在 Flink 中会被映射为 `BINARY(1)`。
+
+OceanBase 数据类型 | Flink SQL 类型 | 描述
```
Comment (rookiegao) [inline docs/content/connectors/oceanbase-cdc(ZH).md:264]: The code of the data type seems different from that of the English document?

Comment (rookiegao) [reply docs/content/connectors/oceanbase-cdc(ZH).md:264]: 英文文档添加了 OceanBase 数据类型的跳转链接

展示效果：
image

源代码：
image

Comment (leonardBang) [reply docs/content/connectors/oceanbase-cdc(ZH).md:262]: > 英文文档添加了 OceanBase 数据类型的跳转链接
> 
> 展示效果： image
> 
> 源代码： image

Have we added this ? I don't think so

Comment (leonardBang) [reply docs/content/connectors/oceanbase-cdc(ZH).md:262]: https://ververica.github.io/flink-cdc-connectors/master/content/connectors/oceanbase-cdc.html#data-type-mapping

Comment (rookiegao) [reply docs/content/connectors/oceanbase-cdc(ZH).md:262]: 该处修改被整合到了 #1719 ，另外 2.3 的所有连接器文档都做出了修改。

参照：
https://ververica.github.io/flink-cdc-connectors/release-2.3/content/connectors/oceanbase-cdc.html#data-type-mapping

**Review by ruanhang1993 — APPROVED** [review]
LGTM.

File: docs/content/connectors/oceanbase-cdc(ZH).md (hunk 2 above)
Comment (leonardBang) [reply docs/content/connectors/oceanbase-cdc(ZH).md:260]: Let's add this in https://github.com/ververica/flink-cdc-connectors/pull/1719

**Pull request merged**